spec/requests/feeds_spec.rb

```diff
@@ -2,15 +2,38 @@
 
 RSpec.describe "/feeds", type: :request do
   let(:valid_attributes) {}
+  let(:feed) { Feed.create! valid_attributes }
 
   describe "GET /show" do
     it "renders a successful response" do
-      feed = Feed.create! valid_attributes
       get feed_url(feed)
       expect(response).to be_successful
     end
   end
 
+  describe "GET .atom" do
+    require "support/assert_valid_feed"
+    include W3C::FeedValidator::Assertions
+
+    context "without a post" do
+      it "is valid according to W3C" do
+        get feed_url(feed, format: :atom)
+        assert_valid_feed
+      end
+    end
+
+    context "with a post" do
+      it "is valid according to W3C" do
+        payload = Rails.root.join("spec", "support", "body.json").read
+        expect {
+          EmailProcessor.for_payload(payload, feed: feed).process
```

Comment:
as cute as forcing the email processor to insert into a chosen feed is, I feel like it will be less code (and more useful as a test!) if the email payload is expected to succeed anyway

Reply:
hmmm I wonder what email address the checked in payload is set to hit

Reply:
``` json
  "To": "somefeed@feedyour.email",
```